cmd/influxd/run.go

```diff
@@ -50,9 +50,14 @@ func Run(config *Config, join, version string, logWriter *os.File) (*messaging.B
 		// We want to make sure we are spun up before we exit this function, so we manually listen and serve
 		listener, err := net.Listen("tcp", config.BrokerAddr())
 		if err != nil {
-			log.Fatal(err)
+			log.Fatalf("failed to listen for broker on tcp: %s ", err)
```

Comment:
Minor suggestion: include the value of `config.BrokerAddr()` in the log message. It might be better phrased as something like "Broker failed to listen on %s: %s" where the first variable is the address, the second is the error. Brokers only listen over TCP, so mentioning TCP isn't strictly necessary.